controllers/tutors.js

```diff
@@ -69,4 +69,34 @@ const signIn = async (req, res) => {
   }
 };
 
-module.exports = { verify, signIn };
+const changePassword = async (req, res) => {
+  
+  try {
+    let tutor = await Tutor.findById(req.params.id);
+    const { newPassword, oldPassword } = req.body;
+    if (await bcrypt.compare(oldPassword, tutor.password_digest)) {
+      const password_digest = await bcrypt.hash(newPassword, SALT_ROUNDS);
+      tutor = await Tutor.findByIdAndUpdate(
+        req.params.id,
+        { password_digest: password_digest },
+        { new: true }
+      );
+      const payload = {
+        id: tutor._id,
+        username: tutor.username,
+        email: user.email,
+      };
+      const token = jwt.sign(payload, TOKEN_KEY);
+      return res.status(201).json({ user, token });
+    }
+    else {
+      return res.status(403).send("wrong password")
```

Comment:
Not super necessary but maybe change the message to "Invalid Password"

Reply:
I will make the changes